ui/src/components/shared/SiderLayoutContent.tsx

```diff
@@ -1,4 +1,4 @@
-import { Button, Col, Divider, Layout, Row, Space, Typography, notification } from "antd";
+import { Button, Col, Divider, Grid, Layout, Row, Typography, notification } from "antd";
```

Comment:
I see this in the console

<img width="1064" alt="image" src="https://github.com/0xPolygonID/issuer-node/assets/15896177/ad66f01f-bc64-4ddf-a41c-cbad29bed919">